test/e2e/deploy/gloo/happy_path_e2e_test.go

```diff
@@ -30,7 +30,7 @@ var _ = Describe("wasme deploy gloo", func() {
 	It("Deploys the filter via Gloo", func() {
 		imageName := test.GetImageTag()
 
-		err := test.WasmeCli("deploy", "gloo", imageName, "--id", "myfilter", "--config", "world")
```

Comment:
Need to figure out how / if we can still support the `--config` flag for newer versions of wasm where config is a protobuf.Any and no longer a string.

Reply:
Addressed in github.com/solo-io/wasme/pull/142